docs/changelog.md

```diff
@@ -1,5 +1,40 @@
 ## Changelog
 
+### v0.6.1 (11/10/2020)
+
+#### Highlights
+
+- Support training with FP16
+- Support docker
+- Update baseline configs and results (CenterPoint on nuScenes and PointPillars on Waymo with full dataset)
+
+#### Bug Fixes
+
+- Fix a bug of visualization in multi-batch case (#120)
+- Fix bugs in dcn unit test (#130)
+- Fix dcn bias bug in centerpoint (#137)
+- Fix dataset mapping in the evaluation of nuScenes mini dataset (#140)
+- Fix origin initialization in cam_box3d (#148, #150)
+- Fix missing page in the `getting started` section of documentation
+- Fix gather_models script (#153)
+
+#### New Features
+
+- Support dataset pipeline `SweepPointSample` (#125)
```

Comment:
Support dataset pipeline `VoxelBasedPointSampler`to sample multi-sweep points based on voxelization.